lib/glific/flows/case.ex

```diff
@@ -83,8 +83,8 @@ defmodule Glific.Flows.Case do
   @doc """
   Validate a case
   """
-  @spec validate(Case.t(), Keyword.t(), map()) :: Keyword.t()
-  def validate(%{arguments: arguments} = _case, errors, flow) when arguments != [] do
+  @spec validate(Case.t(), Keyword.t(), map(), boolean()) :: Keyword.t()
+  def validate(%{arguments: arguments} = _case, errors, flow, false) when arguments != [] do
```

Comment:
We need to apply the validation if has_wait is true right? not false

Reply:
@pankaj-ag made requested changes